docs/src/paradox/03-apis/api-admin/projects.md

```diff
@@ -74,7 +80,20 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
       "selfjoin": false
     }
     ```
-
+    Each project can have an optional custom IRI specified by the `projectIri` in the request body as below:
+    ```JSON
+    {
+        "projectIri": "http://rdfh.ch/projects/3333",
+        "shortname": "newprojectWithIri",
```

Comment:
I don't believe that the format of the custom IRI was documented somewhere in the past. Maybe the format that we expect and validate for could be mentioned somewhere in the docs?

Reply:
Should we require the normal project IRI format?

https://docs.knora.org/paradox/03-apis/api-v2/knora-iris.html#iris-for-data

Reply:
I think so. This way, each project has it's own namespace that they can use however they believe it's best, without interfering with other projects. Also, if we allow any kinds of IRIs, then we would need to remove validation in all places and maybe even change the ARK server.

Reply:
And thanks for the link. We could link to the page from here.